### PR TITLE
Move withdrawal explanation page to the GOV.UK Design System

### PR DESCRIPTION
**Pull request by dnkrj**
⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### What
[Move the withdrawal explaination page to the GOVUK design system layout.
](https://trello.com/c/LhicV17K/776-move-withdrawal-explanation-page-to-the-govuk-design-system)
### Why
This is part of a Whitehall transition work to move from bootstrap to GOVUK design system.

### Screenshots
whitehall-admin dev gov uk_government_admin_editions_5_unpublishing_edit(iPad Pro)
whitehall-admin dev gov uk_government_admin_editions_5_unpublishing(iPad Pro) (1)
whitehall-admin dev gov uk_government_admin_editions_5_unpublishing_edit(iPhone 12 Pro)

#### Before
whitehall-admin dev gov uk_government_admin_editions_1_unpublishing_edit(iPad Pro)
whitehall-admin dev gov uk_government_admin_editions_1_unpublishing_edit(iPhone 12 Pro)

### CODE REVIEW

File: app/views/admin/edition_unpublishing/edit.html.erb [hunk 1]
```diff
@@ -1,31 +1,39 @@
-<% edition = @unpublishing.edition
-   withdrawal_or_unpublishing = withdrawal_or_unpublishing(@unpublishing.edition)
-%>
+<% content_for :context, @unpublishing.edition.title %>
+<% content_for :page_title, "Edit #{withdrawal_or_unpublishing(@unpublishing.edition)} explaination" %>
```
Comment (davidtrussler) [inline app/views/admin/edition_unpublishing/edit.html.erb:2]: I think this is a typo (yeah I know it follows the design) - "explanation" not "explaination"

File: app/views/admin/edition_unpublishing/edit.html.erb [hunk 2]
```diff
@@ -1,31 +1,39 @@
...
-        %>
-      </fieldset>
-      <fieldset>
-        <%= f.submit "Update #{withdrawal_or_unpublishing} explanation", class: "btn btn-primary" %>
-        <%= link_to 'Cancel', [:admin, edition], class: 'btn btn-default add-left-margin' %>
-      </fieldset>
+      <%= render "govuk_publishing_components/components/button", {
+        text: "Update #{withdrawal_or_unpublishing(@unpublishing.edition)} explaination",
+        margin_bottom: 8
+      } %>
```
Comment (davidtrussler) [inline app/views/admin/edition_unpublishing/edit.html.erb:36]: As above on "explaination".

**Comment by davidtrussler** [comment]
Hi @dnkrj - just left a comment about a typo (in three places) but otherwise looks good so can approve when that's done.

**Comment by dnkrj** [comment]
Good catch! Thanks!

**Review by davidtrussler — APPROVED** [review]
LGTM! 👍

File: app/views/admin/edition_unpublishing/edit.html.erb [hunk 3]
```diff
@@ -1,31 +1,39 @@
...
+<% content_for :back_link do %>
+  <%= render "govuk_publishing_components/components/back_link", {
+    href: admin_edition_path(@unpublishing.edition)
+  } %>
+<% end %>
 
-<% page_title "Edit public explanation for #{withdrawal_or_unpublishing}"  %>
+<div class="govuk-grid-row">
+  <section class="govuk-grid-column-two-thirds">
+    <h2 class="govuk-heading-l govuk-!-margin-bottom-3">Reason for <%= withdrawal_or_unpublishing(@unpublishing.edition) %></h2>
```
Comment (DilwoarH) [inline app/views/admin/edition_unpublishing/edit.html.erb:14]:
```suggestion
    <%= render "govuk_publishing_components/components/heading", {
        text: withdrawal_or_unpublishing(@unpublishing.edition),
        font_size: "l",
        margin_bottom: 3
    } %>
```

**Review by DilwoarH — CHANGES_REQUESTED** [review]
Just something very minor

**Review by DilwoarH — APPROVED** [review]
Thanks - looks good

**Pull request merged**